server/webhook_worker.go

```diff
@@ -49,7 +51,30 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 		return err
 	}
 
-	channelsSubscribed, err := ww.p.getChannelsSubscribed(v, msg.InstanceID)
+	// To check if this is a comment-related webhook payload
+	isCommentEvent := wh.Events().Intersection(commentEvents).Len() > 0
```

Comment:
I think a big part of this code can be done as a function/method some place else and call it here, you could then remove the comment if you put a relevant name to the function and it's easier to maintain/test in the future since the function has only one responsibility (Check if this is a comment-related webhook payload).